src/composables/useNumbers.ts

```diff
@@ -1,17 +1,37 @@
 import numeral from 'numeral';
 import BigNumber from 'bignumber.js';
 import useTokens from './useTokens';
+import useUserSettings from '@/composables/useUserSettings';
 
 interface Options {
   format?: string;
   forcePreset?: boolean;
 }
 
+export interface FNumOptions extends Intl.NumberFormatOptions {
+  fixedFormat?: boolean; // If true, don't auto-adjust based on number magnitde
+  abbreviate?: boolean; // If true, reduce number size and add k/M/B to end
+}
+
+export const FNumFormats = {
+  percent: {
+    style: 'unit',
+    unit: 'percent',
+    minimumFractionDigits: 2,
+    maximumFractionDigits: 2
+  },
+  token: {
+    maximumFractionDigits: 4
+  }
+};
```

Comment:
As mentioned above I think we could add a fiat preset:
```suggestion
export const FNumFormats = {
  percent: {
    style: 'unit',
    unit: 'percent',
    minimumFractionDigits: 2,
    maximumFractionDigits: 2
  },
  token: {
    maximumFractionDigits: 4
  },
  fiat: {
    style: 'currency'
  }
};
```